docs/written/trove/lists.scrbl

```diff
@@ -1623,13 +1623,104 @@ end
 }
 
 @list-method["member"]
+
+Returns true if the @pyret{elt} is located somewhere in the list.
+
+@examples{
+check:
+  [list: 'a', 'b', 'c'].member('a') is true
```

Comment:
Yeah, I am actually neutral on this, but recently Shriram suggests that it's a good idea to produce a hint message to use `satisfies` when someone writes `f(x) is true`. If this is going to be implement, I think that we shouldn't have `f(x) is true` in the documentation.